crates/ruff_linter/src/rules/flake8_use_pathlib/rules/replaceable_by_pathlib.rs

```diff
@@ -202,6 +202,9 @@ pub(crate) fn replaceable_by_pathlib(checker: &Checker, call: &ExprCall) {
         ["os", "path", "getmtime"] => Diagnostic::new(OsPathGetmtime, range),
         // PTH205
         ["os", "path", "getctime"] => Diagnostic::new(OsPathGetctime, range),
+        // PTH211
+        ["os", "symlink"] => Diagnostic::new(OsSymlink, range),
```

Comment:
We should avoid raising this rule when `fd` is set to a non-None` value (see chmod for an example)